cn.ottercorp.xivlaunchercn.appdata.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop-application">
+  <id>cn.ottercorp.xivlaunchercn</id>
```

Comment:
this app-id doesn't meet the requirements as `ottercorp.cn` doesn't seem to exist.

Reply:
The @ record of the domain is not assigned since we're using http://xivlauncher.ottercorp.cn/ to distribute.

Reply:
it is not mentionned anywhere. the appstream file mention a github.io as homepage as well.

Reply:
It is not necessary to host the homepage to the application id since according to the [App-Requirements](https://github.com/flathub/flathub/wiki/App-Requirements#application-id) `The Application ID should be a real URL of a domain that the app author has control over or where their app is hosted.` I control the URL/domain, just not hosting the homepage on it.
Besides that, I think the prior XIVLauncher is also hosting its homepage in github.io https://github.com/flathub/flathub/pull/3101/files.

Reply:
> it is not mentionned anywhere. the appstream file mention a github.io as homepage as well.

If you mean "not mentioned anywhere" in the code base, we are actually using `ffxiv.wang` (or `aonyx.ffxiv.wang` as the CDN layer in CN.